lighthouse-core/computed/trace-of-tab.js

```diff
@@ -61,10 +65,10 @@ class TraceOfTab {
       sorted.push(traceEvents[indices[i]]);
     }
 
+    // @ts-ignore - TODO ???
```

Comment:
help? I think the issue is that it's impossible to set up type guards on template params in jsdoc, so I can't require that `T` be of type `TraceEvent`.